apps/studio/components/interfaces/Auth/Policies/PolicyTableRow/index.tsx

```diff
@@ -59,27 +71,37 @@ export const PolicyTableRow = ({
         />
       }
     >
-      {!table.rls_enabled && !isLocked && (
+      {!!rlsLint && (
         <div
           className={cn(
             'dark:bg-alternative-200 bg-surface-200 px-6 py-2 text-xs flex items-center gap-2',
             policies.length === 0 ? '' : 'border-b'
           )}
         >
-          <div className="w-1.5 h-1.5 bg-warning-600 rounded-full" />
-          <span className="font-bold text-warning-600">Warning:</span>{' '}
-          <span className="text-foreground-light">
-            Row Level Security is disabled. Your table is publicly readable and writable.
-          </span>
-          <Tooltip>
-            <TooltipTrigger asChild>
-              <Info className="w-3 h-3" />
-            </TooltipTrigger>
-            <TooltipContent className="w-[400px]">
-              Anyone with the project's anonymous key can modify or delete your data. Enable RLS and
-              create access policies to keep your data secure.
-            </TooltipContent>
-          </Tooltip>
+          <div
+            className={cn(
+              'w-1.5 h-1.5 rounded-full',
+              rlsLint.level === 'ERROR' ? 'bg-warning-600' : 'bg-selection'
+            )}
+          />
+          <span className={cn('font-bold', rlsLint.level === 'ERROR' ? 'text-warning-600' : '')}>
+            {rlsLint.level === 'ERROR' ? 'Warning' : 'Note'}:
+          </span>{' '}
+          <Markdown
+            className="text-xs text-foreground-light !max-w-full"
+            content={`${rlsLint.detail}${rlsLint.level === 'ERROR' ? ' Your table is publicly readable and writable.' : '. No data will be selectable via Supabase APIs.'}`}
```

Comment:
the if condition `rlsLint.level === 'ERROR'` should be about the specific rls rule, not the level.